Automattic/woocommerce-payments
# Migrate get_account_data API to the core Request class

Fixes 
A part of https://github.com/Automattic/woocommerce-payments-meta/issues/87

This PR is going to migrate the first simple API first, then I will migrate subsequent endpoints in other PRs based on this one.

#### Changes proposed in this Pull Request









#### Testing instructions





* Use WCPay Dev tool, and vist the setting page. 
* Click `clear` for `Account cache content`.
* See no error in the PHP error log.

-------------------


**Post merge**
